editor/script_create_dialog.cpp

```diff
@@ -369,27 +369,27 @@ void ScriptCreateDialog::_path_changed(const String &p_path) {
 	String p = p_path.strip_edges();
 
 	if (p == "") {
-		_msg_path_valid(false, TTR("Path is empty"));
+		_msg_path_valid(false, TTR("Path is empty."));
 		_update_dialog();
 		return;
 	}
 
 	if (p.get_file().get_basename() == "") {
-		_msg_path_valid(false, TTR("Filename is empty"));
+		_msg_path_valid(false, TTR("File name is empty."));
```

Comment:
Filename seems to be more common. See for example [this discussion on wikipedia](https://en.wikipedia.org/wiki/Talk:Filename#%22File_name%22_vs_%22Filename%22) (the article there is named "Filename" as well and starts with "A filename (also written as two words, file name)" and [this stackexchange answer](https://english.stackexchange.com/questions/5366/which-is-correct-filename-file-name-or-filename/5374#5374).